2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -147,4 +138,90 @@ sezioni.
 
 === Elenchi puntati
-Viene inserito ";" alla fine delle voci dell'elenco tranne l'ultima che termina con ".". Ogni voce dell'elenco inizia con una lettera minuscola.
\ No newline at end of file
+Viene inserito ";" alla fine delle voci dell'elenco tranne l'ultima che termina con ".". Ogni voce dell'elenco inizia con una lettera minuscola.
+
+= Processi di ciclo di vita
+
+== Processi organizzativi abilitanti
+
+I processi organizzativi abilitanti garantiscono la capacità dell'organizzazione di acquisire e fornire prodotti o servizi attraverso l'avvio, il supporto e il controllo di progetti. Questi processi forniscono le risorse e l'infrastruttura necessarie a supportare i progetti e il conseguimento degli obiettivi dell'organizzazione e degli accordi fra parti. Non sono da intendersi come un insieme esaustivo di processi aziendali atti alla gestione strategica dell'organizzazione.
+
+I processi organizzativi abilitanti sono i seguenti:
+//+ Processo di gestione dei modelli di ciclo di vita;
++ Processo di gestione dell'infrastruttura;
+// + Processo di gestione del portfolio;
+// + Processo di gestione delle risorse umane;
+// + Processo di gestione della qualità;
+// + Processo di gestione della conoscenza.
+
+=== Processi di gestione dell'infrastruttura
+
+==== Scopo
+
+Lo scopo del processo di gestione dell'infrastruttura è fornire l'infrastruttura e i servizi a supporto dell'organizzazione per il conseguimento degli obiettivi di progetto nel corso dell'intero ciclo di vita.
+
+Questo processo definisce, fornisce e regola i servizi, gli strumenti e le tecnologie di comunicazione e condivisione delle informazioni a supporto degli scopi dell'organizzazione.
+
+==== Requisiti
+
+L'infrastruttura è costituita dai servizi, dagli strumenti e dalle tecnologie di comunicazione e condivisione delle informazioni adottate a supporto degli scopi dell'organizzazione. L'infrastruttura risponde alle necessità di comunicazione interna ed esterna.
+
+// Li sposterei
+I principi della comunicazione che ispirano i processi di gestione dell'infrastruttura sono:
+
++ Comunicazione aperta: le comunicazioni avvengono in modalità pubblica, ovvero tutti i membri possono partecipare (compatibilmente con i loro impegni di progetto e sempre nel rispetto delle rispettive responsabilità). I membri del gruppo hanno accesso e possono liberamente consultare i messaggi, le eventuali registrazioni e i verbali;
++ Comunicazione onesta: quanto comunicato rappresenta sempre il reale stato del progetto. Nessun membro, in nessun caso, deve nascondere le criticità incontrate.
++ Comunicazione proattiva: comunicare con cognizione di causa, offrendo spunti concreti di discussione. Ogni comunicazione deve poter essere processata dagli interessati nel minor tempo possibile.
```

Comment:
Sostituire il punto con un punto e virgola alla fine delle righe 173 e 174

Reply:
Sistemato. Secondo te ha senso questa sezione di "principi"? Secondo me potrebbe aver senso, ma non mi sembra che questa sezione sia il posto giusto

Reply:
Secondo me ha senso e può anche stare qui. Altrimenti se preferisci si potrebbe fare una cosa del tipo `==== Principi` e inserirli lì, o rinominare la sezione in `Principi e requisiti`

Reply:
Ho corretto (ancora due giorni fa). Credo che tu debba rilasciare nuovamente una review